pyccel/codegen/python_wrapper.py

```diff
@@ -119,7 +119,7 @@ def create_shared_library(codegen,
 
         setup_code = create_c_setup(sharedlib_modname, wrapper_filename,
                 dep_mods, compiler, includes, libs + extra_libs, libdirs + extra_libdirs, c_flags)
-        setup_filename = "setup_{}.py".format(module_name)
```

Comment:
Why changing the name of the setup file, if the name of the bind_c and wrapper files are not changed?

Reply:
I can't remember why I changed this, I have changed it back. See fee9773